packages/react/src/components/TextField/TextField.test.tsx

```diff
@@ -731,81 +695,79 @@ describe('TextField onChange', () => {
   function simulateAndVerifyChange(changeValue: string, calls: number, expectedValue?: string) {
     expectedValue = typeof expectedValue === 'string' ? expectedValue : changeValue;
 
-    const input = wrapper!.find('input');
-    // Fire input AND change events to more realistically test
-    input.simulate('input', mockEvent(changeValue));
-    input.simulate('change', mockEvent(changeValue));
+    const input = getInput();
+    fireEvent.change(input, { target: { value: changeValue } });
```

Comment:
Went with using `fireEvent.change` here to maintain the initial behavior of the function and to prevent having to update the tests that use it. Using `userEvent.type` leads to an `onChange` event being triggered for each character in the string.

Reply:
That's interesting...in reality if the user typed it *would* fire a change event each time, so the test is more like if they pasted a value. But if (effectively) pasting a value is still testing the intended behavior in a realistic enough way, that could be okay.